cmd/controller/main.go

```diff
@@ -37,6 +38,11 @@ import (
 	controllerruntimezap "sigs.k8s.io/controller-runtime/pkg/log/zap"
 )
 
+const (
+	maxBatchWindow   = 10 * time.Second
```

Comment:
Can we hide these values inside the allocation.NewController function and just manually instantiate the struct for our testing? I think main shouldn't have this awareness.

Reply:
Yeah, I think that can work.

Reply:
Oh actually, we'd have to make the Controller fields public for that to work, since the aws cloudprovider suite tests are not in the same package.

Reply:
Public sounds fine to me. Better than forcing this awareness to this file.